extensions/wikidata/module/scripts/dialogs/schema-alignment-dialog.js

```diff
@@ -148,10 +145,16 @@ SchemaAlignmentDialog.setUpTabs = function() {
 
   this._previewPanes = $(".schema-alignment-dialog-preview");
 
-  // Load the existing schema
-  this._reset(theProject.overlayModels.wikibaseSchema);
-  // Perform initial preview
-  this.preview();
+  var url = ReconciliationManager.ensureDefaultServicePresent();
```

Comment:
This is depending on Wikidata being the default reconciliation service which is true today (and this patch doesn't change this aspect of the behavior), but it seems like the Wikidata extension should know the address of its own reconciliation service.

Also, using the ensureDefaultServicePresent function to fetch this seems rather roundabout.